dallinger/models.py

```diff
@@ -59,6 +59,11 @@ class SharedMixin(object):
     #: a generic column for storing structured JSON data
     details = Column(JSONB, nullable=False, server_default="{}", default=lambda: {})
 
+    #: HTML string to display in visualizations (e.g. the Network
+    #: Montioring Dashboard). You can override this with a dynamic ``@property``
+    #: on sub-classes.
+    visualization_html = ""
```

Comment:
I'm a little unclear what effect customizing this would have. I expected to see a "nothing" somewhere when running the demo, as a consequence of the default "" implementation, but I don't. Does the current JS just not use it?

Reply:
It's used. If your custom Node class or whatever defines something like:

    @property
    def visualization_html(self):
        return "some custom string with dynamic values"

It will be rendered in the "Details" section of the monitor when selecting a node. The empty string renders as an empty div in the html.